JSDemos/Demos/DataGrid/ExportToPDF/Angular/app/app.component.html

```diff
@@ -1,15 +1,23 @@
-<dx-button
+<dx-button id="exportButton"
```

Comment:
Why do you set the id attribute for the button?

Reply:
I use the ID of the button to set the CSS style for it.
The bottom margin, in the 'app.component.css' file.